### PR TITLE
Added repo class

### PR DESCRIPTION
**Pull request by Kugelsicher**

### CODE REVIEW

**Review by nicholasvvong — APPROVED** [review]
Seems to be missing repo changes? Adding to check

File: Repository/Models/FollowingMovie.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+
+#nullable disable
+
+namespace Repository.Models
+{
+    public partial class FollowingMovie
+    {
+        public string Username { get; set; }
+        public string MovieId { get; set; }
```
Comment (markmoorerev) [inline Repository/Models/FollowingMovie.cs:11]: is this the string you get from the API? If not, why is it a string?

File: Repository/Models/FollowingUser.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+using System.Collections.Generic;
+
+#nullable disable
+
+namespace Repository.Models
+{
+    public partial class FollowingUser
+    {
+        public string Follower { get; set; }
+        public string Followee { get; set; }
```
Comment (markmoorerev) [inline Repository/Models/FollowingUser.cs:11]: make these more descriptive. like 'FollowerId', 'FollowerName', etc

File: Repository/RepoLogic.cs [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+        /// <summary>
+        /// Returns a list of all FollowingMovie objects from the database that match the username
+        /// specified in the argument. Returns null if the user doesn't exist.
+        /// </summary>
+        /// <param name="username"></param>
+        /// <returns></returns>
+        public async Task<List<FollowingMovie>> GetFollowingMovies(string username)
+        {
+            var userExists = UserExists(username);
```
Comment (markmoorerev) [inline Repository/RepoLogic.cs:91]: Is username guaranteed to be unique?

File: Repository/RepoLogic.cs [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+            await _dbContext.SaveChangesAsync();
+            return true;
+        }
+
+        /// <summary>
+        /// Returns true iff the username, specified in the argument, exists in the database's Users table.
+        /// </summary>
+        /// <param name="username"></param>
+        /// <returns></returns>
+        private bool UserExists(string username)
```
Comment (markmoorerev) [inline Repository/RepoLogic.cs:140]: Is this method used for logging in? If so, you should be using a combo username and password to find the user. Of course, separate methods for unique username and unique password would be more reusable.

**Review by markmoorerev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**